test/mocks/PsmMock.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.16;
+
+interface GemLike {
+    function approve(address spender, uint256 value) external;
+    function transfer(address, uint256) external;
+    function transferFrom(address, address, uint256) external;
+    function decimals() external view returns (uint8);
+}
+
+contract KegMock {
```

Comment:
```suggestion
contract PocketMock {
```

Reply:
I'd actually argue that for the Mock you do not even need the `Pocket`.

Reply:
Agreed, I removed it from the mock.